src/main/resources/application.properties

```diff
@@ -15,4 +15,4 @@ activity.daily-status.repositories.quarkus-extensions-combinations=https://githu
 activity.daily-status.repositories.quarkus-startstop=https://github.com/quarkus-qe/quarkus-startstop/actions/workflows/ci.yaml
 activity.daily-status.repositories.quarkus-test-framework=https://github.com/quarkus-qe/quarkus-test-framework/actions/workflows/daily.yaml
 activity.daily-status.repositories.quarkus-test-suite=https://github.com/quarkus-qe/quarkus-test-suite/actions/workflows/daily.yaml
-activity.daily-status.repositories.quarkus-test-suite-java-17=https://github.com/quarkus-qe/quarkus-jdkspecifics/actions/workflows/ci.yml
\ No newline at end of file
+activity.daily-status.repositories.quarkus-test-suite-jdk-specific=https://github.com/quarkus-qe/quarkus-jdkspecifics/actions/workflows/ci.yml
```

Comment:
Maybe `activity.daily-status.repositories.quarkus-test-suite-jdkspecifics` would be better?

Reply:
Updated